packages/contract/README.md

```diff
@@ -80,51 +79,47 @@ v4.0.2 or higher.
 Let's use `@truffle/contract` with an example contract from [Dapps For Beginners](https://dappsforbeginners.wordpress.com/tutorials/your-first-dapp/). In this case, the abstraction has been saved to a `.sol` file by [@truffle/artifactor](https://github.com/trufflesuite/truffle/tree/master/packages/artifactor):
 
 ```javascript
+const contract = require("@truffle/contract");
 // Require the package that was previosly saved by @truffle/artifactor
-var MetaCoin = require("./path/to/MetaCoin.json");
+const metacoinArtifact = require("./path/to/MetaCoin.json");
+const metacoin = contract(metacoinArtifact)
```

Comment:
Happy to change these variable names if anyone has alternatives

Reply:
I think using a capital would be more in line with convention here, since the artifact represents a class rather than an instance.

Reply:
Yeah, `MetaCoin` for the Truffle Contract object and something like `instance` for the deployed instance is convention I think.

Reply:
👍